src/audio/wav64.c

```diff
@@ -362,5 +375,8 @@ void wav64_close(wav64_t *wav)
 			break;
 		}
 		wav->ext = NULL;
+		if (wav->currentFile >= 0) {
+			close(wav->currentFile);
```

Comment:
Please also invalidate the variable by setting it to -1. It helps with diagnostics if the user tries to use the wav object again by mistake.